reference/src/layout/enums.md

```diff
@@ -313,13 +317,21 @@ values, which are called **niches**. For example, a value of type `&T`
 may never be `NULL`, and hence defines a niche consisting of the
 bitstring `0`.  Similarly, the standard library types [`NonZeroU8`]
 and friends may never be zero, and hence also define the value of `0`
-as a niche. (Types that define niche values will say so as part of the
-description of their validity invariant, which -- as of this writing
--- are the next topic up for discussion in the unsafe code guidelines
-process.)
+as a niche. 
 
 [`NonZeroU8`]: https://doc.rust-lang.org/std/num/struct.NonZeroU8.html
 
+The niche values of a type are parts of its validity invariant which, as of this
+writing, is the current active discussion topic in the unsafe code guidelines
+process. [rust-lang/rust#60300] specifies that the following types have a niche:
```

Comment:
Should this maybe say what the niche is? I think we technically only fixed a lower bound, but still.

Reply:
There is a TODO in the glossary for that.

Reply:
No I don't mean define what the term niche means. I mean say concretely something like "The niche of `NonZeroU8` is the value 0".

Reply:
Ah, I misunderstood. We can try to explicitly say that, e.g., for `NonZero`, `NonNull`, etc. But as you mentioned elsewere this might be harder for `&T`, where zero is definitely a niche, but there might be others, and those are currently unclear.

Reply:
We can say the niche includes at least "0". That's probably something we want to guarantee.

That's why I said we only have a lower bound.

Reply:
Fixed